### PR TITLE
update test and getValues method to url values

### PR DESCRIPTION
**Pull request by ros3bud**
@zalary, this potentially solves issue #14

- update the test body in add_item_spec.coffee
- update method calls with parenthesis for consistency in query_item.coffee
- utilize escape method to URL encode in helper.coffee

### CODE REVIEW

**Comment by ros3bud** [comment]
Ok, noted. Looking into this.

**Comment by ros3bud** [comment]
- Using encodeURIComponent() rather than escape() will encode everything with the exception of:
~!*()'

File: spec/add_item_spec.coffee [hunk 1]
```diff
@@ -16,7 +16,7 @@ describe 'Add List Item', ->
       assert.equal 'POST', request.method
 
     it 'should have the correct body', ->
-      assert.equal '{"values":"boilermakers@example.com|taylor@activeprospect.com"}', request.body
+      assert.equal '{"values":"boilermakers@example.com%2C%20taylor@activeprospect.com"}', request.body
```
Comment (alexkwolfe) [inline spec/add_item_spec.coffee]: It is not necessary to URL encode JSON.

**Comment by ros3bud** [comment]
@alexkwolfe - so this change is not needed?

**Comment by alexkwolfe** [comment]
It's not clear to me what problem issue #14 is reporting...

**Comment by ros3bud** [comment]
Ok, I will talk with Zalary to get more details.

**Comment by cgrayson** [comment]
Noticed this old PR still hanging out, @ros3bud (or @zalary) - do you know if this is still a pending thing?

**Comment by ros3bud** [comment]
@cgrayson - it is still pending. Here is what needs to happen: 

1. Send test leads to a flow which uses SuppressionList in LC which contains emails with odd characters.  For example: #1&2%3.

2. Once it's determined how the odd characters are handled, update the integration accordingly.

**Comment by ros3bud** [comment]
Going to close out this PR and add a new issue as this has taken a new shape.

**Pull request closed without merging**